src/styles/blocks/page.scss

```diff
@@ -0,0 +1,44 @@
+html {
+  width: 100%;
+  scroll-behavior: smooth;
+  overflow: hidden auto;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #F2F6FA;
+  font-family: Poppins, sans-serif;
+  max-width: 100vw;
+
+  &:has(.menu:target) {
+    /* not to scroll the page */
+    overflow: hidden;
+  }
+}
+
+.btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #2060F6;
+  color: #fff;
+  font-size: 15px;
+  font-weight: 600;
+  
+  border: none;
+  border-radius: 8px;
+  width: 280px;
+  height: 50px;
+  margin-bottom: 48px;
+  padding: 0;
+
+  cursor: pointer;
+
+  &:hover {
```

Comment:
Hover transitions should be smooth both entering and leaving hover; this issues doesn't happen just here; please double check every hover element for usability